### ISSUE TITLE
Suspected Memory Leak

### ISSUE THREAD

**Issue by theelderbeever** [davidB/tracing-opentelemetry-instrumentation-sdk#87]
I am using your `init_trace_subscribers` and sdk and while running there is a steady memory usage growth with them enabled and sending traces. After my api runs for ~7 days servicing about 100 transaction per minute during that time the application OOMs and restarts. If I turn off tracing the memory stays flat at a low value. Not sure if you have run into this before. I am going to try some memory profiling once I have access to a computer I can run profiling with.

**Comment by davidB** [comment]
Thanks for the report, I'm curious to learn more about your future profiling and to help to fix it.

**Comment by theelderbeever** [comment]
@davidB Yep can do and will report back. Hoping to do the profiling this week!

**Comment by davidB** [comment]
to reopen, if reproducible and some metrics
